app/components/Home.js

```diff
@@ -0,0 +1,675 @@
+                      rating: 4.40,
+                      drinkNumber: 81,
+                      url: 'http://www.barmeister.com/drinks/recipe/1142/?o=top100'
+                    },
+                    {
+                      name: 'Hunch Punch',
+                      ingredients: '25 oz Vodka, 25 oz Whiskey, 25 oz Rum, 1 package Kool-Aid dissolved in water. Serve with or without fruit. Be careful. ',
+                      rating: 4.39,
+                      drinkNumber: 82,
+                      url: 'http://www.barmeister.com/drinks/recipe/780/?o=top100'
+                    },
+                    {
+                      name: 'Cape Cod',
+                      ingredients: '1 part Vodka, 2 part Cranberry Juice',
+                      rating: 4.39,
+                      drinkNumber: 83,
+                      url: 'http://www.barmeister.com/drinks/recipe/297/?o=top100'
+                    },
+                    {
+                      name: 'Red Devil',
+                      ingredients: '1 oz Sloe Gin, 1/2 oz Southern Comfort Whiskey, 1/2 oz Amaretto, 1/2 oz Vodka, 1/2 oz Lime Juice, 1/2 oz Triple Sec, 5 oz Orange Juice',
+                      rating: 4.38,
+                      drinkNumber: 84,
+                      url: 'http://www.barmeister.com/drinks/recipe/1251/?o=top100'
+                    },
+                    {
+                      name: 'Gladiator',
+                      ingredients: '1/2 oz Southern Comfort Whiskey, 1/2 oz Amaretto, 1 part 7-Up, 1 part Orange Juice',
+                      rating: 4.38,
+                      drinkNumber: 85,
+                      url: 'http://www.barmeister.com/drinks/recipe/672/?o=top100'
+                    },
+                    {
+                      name: 'Lemon Drop',
+                      ingredients: '1 oz Vodka, a pinch of Sugar, a slice of Lemon',
+                      rating: 4.35,
+                      drinkNumber: 86,
+                      url: 'http://www.barmeister.com/drinks/recipe/898/?o=top100'
+                    },
+                    {
+                      name: 'Juicy Pussy',
+                      ingredients: '3/4 oz Peach Schnapps, 1/4 oz Bailey\'s Irish Cream, 1/2 glass of Orange Juice',
+                      rating: 4.36,
+                      drinkNumber: 87,
+                      url: 'http://www.barmeister.com/drinks/recipe/853/?o=top100'
+                    },
+                    {
+                      name: 'Singapore Sling',
+                      ingredients: '1 1/2 oz Gin, 1/2 oz Brandy, 3 dashes Benedictine, dash of Grenadine, dash of Lemon Juice, fill with Soda Water',
+                      rating: 4.37,
+                      drinkNumber: 88,
+                      url: 'http://www.barmeister.com/drinks/recipe/1393/?o=top100'
+                    }
+                  ],
+                };
+    },
+
+    getLocalStorage: function () {
```

Comment:
this is not technically 'getLocalStorage' its more like 'getDrinkListFromLocalStorage' a better method would be `getLocalStorage(key)` and use the key to retrieve any object from local storage